chaoscenter/authentication/api/routes/project_router.go

```diff
@@ -18,7 +18,7 @@ func ProjectRouter(router *gin.Engine, service services.ApplicationService) {
 	router.GET("/get_project_role/:project_id", rest.GetProjectRole(service))
 	router.GET("/list_projects", rest.GetProjectsByUserID(service))
 	router.GET("/get_projects_stats", rest.GetProjectStats(service))
-	router.GET("/list_invitations", rest.ListInvitations(service))
+	router.GET("/list_invitations_with_filters/:invitation_state", rest.ListInvitations(service))
```

Comment:
@Saranya-jena as per standard naming convention, dashes (-) are used instead of underscore in url params. 
`invitation_state` can be updated to `invitation-state`